src/transport_ofi.c

```diff
@@ -292,13 +292,6 @@ static inline int allocate_endpoints(struct fabric_info *info)
 
     /* this endpoint is used only to get read and write
      * counter updates */
-    info->p_info->caps = FI_RMA | FI_WRITE | FI_READ | /*SEND ONLY */
-                         FI_ATOMICS; /* request atomics capability */
-    info->p_info->caps |= FI_REMOTE_WRITE | FI_REMOTE_READ;
-    info->p_info->tx_attr->op_flags = FI_DELIVERY_COMPLETE | FI_INJECT_COMPLETE;
```

Comment:
Implicitly, FI_RMA == FI_READ | FI_WRITE | FI_REMOTE_WRITE | FI_REMOTE_READ. Same with FI_ATOMIC. Therefore, the explicit addition of flags in this routine is excessive, but not incorrect.

I think the intent was to disable the RX side caps from the domain. If so, then just remove line 297 and it will be fine.